third_party/conan/docker/build_containers.sh

```diff
@@ -32,10 +32,11 @@ declare -A profile_to_dockerfile=( \
   ["msvc2019_debug"]="msvc2019" \
   ["msvc2019_release_x86"]="msvc2019" \
   ["msvc2019_relwithdebinfo_x86"]="msvc2019" \
-  ["msvc2019_debug_x86"]="msvc2019" )
+  ["msvc2019_debug_x86"]="msvc2019" \
+  ["clang_format"]="clang_format")
```

Comment:
Is there a space at the end right before the parenthesis missing?

Reply:
Maybe. I will add one. Thanks!

Reply:
Done.